lib/v2/141ppv/index.js

```diff
@@ -39,7 +39,10 @@ module.exports = async (ctx) => {
                 .map((t) => $(t).text().trim());
             const magnet = item.find('a[title="Magnet torrent"]').attr('href');
             const link = item.find('a[title="Download .torrent"]').attr('href');
-            const image = item.find('.image').attr('src');
+            const onErrorAttr = item.find('.image').attr('onerror');
+            const backupImageRegex = /this\.src='(.*?)'/;
+            const match = backupImageRegex.exec(onErrorAttr);
+            const image = match ? match[1] : null;
```

Comment:
The old img url can be used as a fallback in case it works.
```suggestion
            const image = match ? match[1] : item.find('.image').attr('src');
```